emission/net/ext_service/habitica/sync_habitica.py

```diff
@@ -0,0 +1,68 @@
+# Standard imports
+import json
+import requests
+import logging
+import arrow
+import attrdict as ad
+
+# Our imports
+import emission.core.get_database as edb
+import emission.net.ext_service.habitica.proxy as proxy
+import emission.analysis.result.metrics.simple_metrics as earmts
+import emission.analysis.result.metrics.time_grouping as earmt
+
+
+
+def create_habit(user_id, new_habit):
+  method_uri = "/api/v3/tasks/user"
+  #how to pass in params? https://habitica.com/apidoc/#api-Task-GetUserTasks
+  #FIX! right now it's returning all types of tasks, I want it to return only "habits"
+  params = {'type': "habits"}
```

Comment:
I didn't try the real habitica server but I did try making direct request calls and that didn't work